guides/v2.1/contributor-guide/contributing_dod.md

```diff
@@ -15,12 +15,20 @@ At a high level, the criteria defined in our "definition of done" revolve around
 
 The following sections provide additional details about each of these criteria:
 
-## Functional Backward Compatibility
+## Backward Compatibility 
+
+### Functional Backward Compatibility
 
 Existing product features and functionality must be retained during any changes to the code.
-If backward-incompatible functional changes are made, the documentation should explain the justification and provide the "business value".
+Any backward-incompatible functional changes must be approbved by a product owner.
+The documentation should explain the justification and provide the "business value".
+
+### Technical Backward Compatibility
+
+Technical backward compatibility must be retained between PATCH versions of Magento products. It should also be retained between MINOR releases if possible.
```

Comment:
Why is MINOR downgraded to "should" instead of "must"?  SemVer rules (which we say we follow) state that all incompatible changes are MAJOR.  If this is referring to marketing version instead of module version it should be clarified (since we allow MAJOR-level module changes in MINOR-level marketing releases; e.g. module goes from 100.x.x to 101.0.0 between Magento 2.2 and 2.3).

Reply:
Yes, it's about marketing version. That's why "versions of Magento products" in the first sentence. But I'll rephrase to make all more clear.